assets/blocks/learner-courses-block/learner-courses-edit.js

```diff
@@ -114,7 +114,17 @@ const LearnerCoursesEdit = ( {
 
 	return (
 		<>
-			<section className={ className }>
+			<section
+				className={ className }
+				style={ {
+					...( undefined !== options.progressBarHeight && {
+						'--progress-bar-height': `${ options.progressBarHeight }px`,
```

Comment:
I used CSS variables to defined the settings styles, so we are more prepared to the Global Styles in the future, and the user could easily override it through CSS without `!important`